translation-web/src/main/java/eu/europeana/api/translation/web/exception/GlobalExceptionHandler.java

```diff
@@ -59,4 +63,23 @@ public ResponseEntity<EuropeanaApiErrorResponse> handleRequestBodyNotParsableErr
   public I18nService getI18nService() {
     return i18nService;
   }
-}
+
+
+  @ExceptionHandler(JsonValidationFailedException.class)
```

Comment:
handler for the JsonValidationFailedException class should be added in EuropeanaGlobalExceptionHandler in api-commons. 
This way this can be handled by any API using JSON schema validation.